storage/myisam/CMakeLists.txt

```diff
@@ -61,6 +67,12 @@ IF(WITH_UNIT_TESTS)
 
   ADD_EXECUTABLE(rt_test rt_test.c)
   TARGET_LINK_LIBRARIES(rt_test myisam)
+
+  IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+    FOREACH(target mi_test1 mi_test2 mi_test3 sp_test rt_test)
```

Comment:
those executables are dead weight.

Reply:
Same than for maria dead executables: they are compiled if we activate unit tests. Without -berok, build fails. Or we must compile without unittest.